fxa-auth-db-server/docs/API.md

```diff
@@ -680,6 +683,56 @@ Content-Type: application/json
 * Status Code : 200 OK
     * Content-Type : 'application/json'
     * Body : `[ ... <see example> ...]`
+* Status Code : 409 Conflict
```

Comment:
`409` is a good code for this from a restfulness standpoint, but I think it would be out-of-place in the auth-server API which tends to use a plain `400` for just about everything that's not specifically covered by a broad `4XX` code convention.  API consumers may find themselves wondering why other types of error don't also generate a `409`.